newton/tests/smoketest_example_browser.py

```
@@ -0,0 +1,152 @@
+        try:
+            viewer.clear_model()
+            mod = importlib.import_module(module_path)
+            ex_parser = getattr(mod.Example, "create_parser", create_parser)()
+            args = default_args(ex_parser)
+            t0 = time.perf_counter()
+            example = mod.Example(viewer, args)
+            _step_and_render(example, cli_args.frames)
+            dt = time.perf_counter() - t0
+            entry["switch"] = "OK"
+            print(f"  switch: OK ({dt:.2f}s)", flush=True)
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)